sdk/metricsadvisor/azure-ai-metricsadvisor-perf/src/main/java/com/azure/ai/metricsadvisor/perf/AnomaliesListTest.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.metricsadvisor.perf;
+
+import com.azure.ai.metricsadvisor.models.ListAnomaliesAlertedOptions;
+import com.azure.ai.metricsadvisor.perf.core.ServiceTest;
+import com.azure.core.util.Context;
+import com.azure.perf.test.core.PerfStressOptions;
+import reactor.core.publisher.Mono;
+
+/**
+ * Performs anomalies list operations.
+ */
+public class AnomaliesListTest extends ServiceTest<PerfStressOptions> {
+    private static final int MAX_LIST_ELEMENTS = 10;
```

Comment:
Should this be made into a passable parameter with a default of 10?

Reply:
let's make it configurable, updating the pr.